code/datums/emotes.dm

```diff
@@ -112,13 +114,15 @@
 	var/dchatmsg = "<b>[user]</b> [msg]"
 
 	var/tmp_sound = get_sound(user)
+	var/emote_volume
+	emote_volume = get_emote_volume(user)
 	if(tmp_sound && should_play_sound(user, intentional) && !TIMER_COOLDOWN_CHECK(user, type))
 		TIMER_COOLDOWN_START(user, type, audio_cooldown)
 		var/tmp_vary = should_vary(user)
 		playsound(
 			source = user,
 			soundin = tmp_sound,
-			vol = 50,
+			vol = emote_volume,
```

Comment:
Thinking about this again. 

Just delete the var completely

Just put get_emote_volume(user) directly in the play sound proc